### PR TITLE
Add music player keybindings

### PR DESCRIPTION
**Pull request by Duarte-Dias**
I have media keys on my laptop and decided to implement their functionality. I used [playerctl](https://github.com/altdesktop/playerctl) since it is simple and seems to be available on a lot of systems.

### CODE REVIEW

File: desktop-environment.el [hunk 1]
```diff
@@ -412,7 +444,10 @@ the screen."
            ;; Wifi controls
            (,(kbd "<XF86WLAN>") . ,(function desktop-environment-toggle-wifi))
            ;; Bluetooth controls
-           (,(kbd "<XF86Bluetooth>") . ,(function desktop-environment-toggle-bluetooth))))
+           (,(kbd "<XF86Bluetooth>") . ,(function desktop-environment-toggle-bluetooth))
+           (,(kbd "<XF86AudioPlay>") . ,(function desktop-environment-toggle-music))
```
Comment (DamienCassou) [inline desktop-environment.el:451]: can you please add a "Music controls" comment to separate the new lines?

**Review by DamienCassou — CHANGES_REQUESTED** [review]
Excellent job. I left a few comments that I would like you to take into consideration. When you are done, please squash all your commits into 1.

**Comment by DamienCassou** [comment]
Thank you very much for your work!

**Comment by Duarte-Dias** [comment]
I made the changes you suggested.
I'm leaving the commits unsquashed so that you can see the latest changes, I believe you can merge with the squashing option. If you can't just ping me again and I will squash.

**Comment by DamienCassou** [comment]
I squashed your commits and merged into `master`. Thank you for your work!

**Pull request closed without merging**